src/field_specials.c

```diff
@@ -1077,36 +1088,35 @@ static void PCTurnOnEffect_0(struct Task *task)
     task->data[3]++;
 }
 
-// enum pc location, 
-static void PCTurnOnEffect_1(s16 flag, s8 dx, s8 dy)
+static void PCTurnOnEffect_1(s16 pcOff, s8 dx, s8 dy)
 {
     u16 tileId = 0;
-    if (flag != 0)
+    if (pcOff != 0)
```

Comment:
`if (isPCTurnedOff)`

Reply:
The first `if` block is to turn the pc off, which occurs if the pc is already on. So changed to `if (isPcTurnedOn)`